### ISSUE TITLE
Multi-dataset fitting interface: optionally display error bars for the data curve

### ISSUE THREAD

**Issue by mantid-roman** [mantidproject/mantid#11318]
This issue was originally [TRAC 10476](http://trac.mantidproject.org/mantid/ticket/10476)

This ticket is blocked by : 
 - [X] [TRAC8911](http://trac.mantidproject.org/mantid/ticket/8911)

This ticket is blocks :  [TRAC11088](http://trac.mantidproject.org/mantid/ticket/11088)

There should be a switch to toggle error bars of the data curve.

**Comment by rosswhitfield** [comment]
The issues will automatically be closed once the pull request is merged into master.

**Comment by mantid-roman** [comment]
I thought "fixes #..." in the pull request closes them. But it didn't

**Comment by rosswhitfield** [comment]
It will when the pull request gets merged.
